### DISCUSSION TITLE
projects beta: non-issue item with body text doesn't indicate presence of body text

### DISCUSSION THREAD

**Discussion by newtmitch** [community/community#10439] [Projects and Issues]
I just added a body to an item that isn't an issue (that was surprising - great feature!) but after saving the body, there's no indication on the board card nor in the list view that a given item has body text at all. Trello indicates the presence of a description with a small icon on the card, it's a great way to make sure that detail put into a given item isn't easily missed.

**Comment by Gabixel** [comment]
<sub>I apologize for the sudden bump, but I found this discussion instead of posting one from scratch and avoiding duplicates. It's also very old...</sub>

I use the board view, which is very simple and perfect, except for this exact tiny detail!

There is no visual way to know if you need to open a card to check additional details, only to find out that it might not even contain a description.

I've made a simple mockup in the following screenshot:

![board view with two cards, one containing an icon indicating the presence of a description](https://github.com/community/community/assets/43073074/d0b4d556-95c4-43d3-bd65-178be8cc01c5)

I really hope they will consider adding this, so we don't forget descriptions.

(Sure, it could also be accomplished by hand, perhaps by assigning a tag, but it's a tedious process, too repetitive, and takes up space for other tags)
![image](https://github.com/community/community/assets/43073074/f30d21a1-cecf-4031-b093-1822cd28cf10)
